config.env

```diff
@@ -0,0 +1,4 @@
+# Configuration for reproschema-ui version
+# Using specific commit hash for stability (from July 27, 2025)
+# You can update this to a different commit hash from https://github.com/ReproNim/reproschema-ui/commits/main
+REPROSCHEMA_UI_CHECKSUM=9f7b6b961cdcb516531bbbf1bf1b74648089d04a
```

Comment:
since the only way to build the UI  is with the github action, you might as well put the env var for the checksum directly in the github action

Reply:
we can keep this one separate, just in case we may want to change something manually